guidelines/program-guidelines.md

```diff
@@ -0,0 +1,81 @@
+# Program Guidelines
+
+## Contributors
+
+- Ashley
+- Jorge
+- Ricky
+- Manish
+
+## Have a clear audience goal
+
+Before you review talks, have a clear idea of what your target audience is. This is not just “users of the technology the conference is about”, this is also includes subgroups by field or level of expertise. Beginner talks will bore advanced users, and advanced talks will not be useful for beginners. When you have multiple rooms, having multiple targets is more feasible; however this can be tricky to schedule, so be careful.
+
+## Introductions
+
+If you have an emcee, be sure to communicate with speakers about this well in advance. Ask how they’d like to be introduced — some may provide you with text, some may care less (in which case, prepare the text yourself and make sure the emcee has it well in advance). Make sure you know their pronouns, and have your emcee work with them to make sure their name will be pronounced correctly.
```

Comment:
`emcee` - I haven't seen it written as this, only as `MC`. Maybe make it more clear who that is and what they are for?